src/commands.rs

```diff
@@ -2415,6 +2404,30 @@ fn cmd_resolve(
     workspace_command.finish_transaction(ui, tx)
 }
 
+fn resolve_list(
```

Comment:
nit: Adding a `print_`/`show_`/`display_` to the name might help clarify that this is about producing output (I first thought it would return a list of conflicts to resolve). I'd probably call it `print_conflicts()`.

Reply:
Good idea. How about `print_conflicted_files`? (As opposed to conflicts within a file)

Reply:
Sounds good. Thanks.

Reply:
I suppose s/files/paths/ is technically more correct, but `print_conflicted_files` also seems fine.

Reply:
I belatedly saw this suggestion and I agree that's a bit better. I created a commit in the other PR, not to forget.